guides/integrate-nextauth.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Integrate Beyond Identity Passwordless Authentication into NextAuth
+sidebar_position: 1
+---
```

Comment:
It also looks like the sidebar_positions are off for everything except the index and enrollment emails. They aren't showing up on the side maybe because they all have the same position number?

Reply:
same here: https://github.com/gobeyondidentity/developer-docs/pull/200